examples/deck/package.json

```diff
@@ -4,8 +4,7 @@
     "start-local": "webpack-dev-server --env.local --progress --hot --port 3000 --open"
   },
   "dependencies": {
-    "@deck.gl/experimental-layers": "5.X.X",
-    "deck.gl": "5.X.X",
+    "deck.gl": "6.0.0-beta.1",
```

Comment:
You should import the experimental layers. Then provide an option to use directed+shadowed for lines.

Reply:
#26